Airports/EGGW/Stars.txt

```diff
@@ -23,6 +23,10 @@ STAR:EGGW:07:UNDUG1N:UNDUG MAY VATON OZZOT BPK ILLOC OXDUF COCCU JUMZI ZAGZO
 STAR:EGGW:25:XAMAN1N:XAMAN IDESI WIQID MUCTE OFJES UDDIZ COCCU JUMZI ZAGZO
 STAR:EGGW:07:XAMAN1N:XAMAN IDESI WIQID MUCTE OFJES UDDIZ COCCU JUMZI ZAGZO
 
+;Withdrawn with LD101 (AIRAC 2303)
+STAR:EGGW:25:#BEDEK1N:BEDEK NIGIT MOREZ VATON OZZOT BPK ILLOC OXDUF COCCU JUMZI ZAGZO
+STAR:EGGW:07:#BEDEK1N:BEDEK NIGIT MOREZ VATON OZZOT BPK ILLOC OXDUF COCCU JUMZI ZAGZO
+
 ;Withdrawn 2022/02
 STAR:EGGW:25:oldAVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
```

Comment:
I know this wasn't this PR, but should these really say "oldAVANT1L" rather than "#AVANT1L"?

Reply:
Because if the STAR starts with the same point, you can't actually select it in the list. So we created these so that if people only had the LOREL STARs they could still be selected on ES.